docs/calicoctl/endpoint.md

```diff
@@ -0,0 +1,191 @@
+>
+# User guide for 'calicoctl endpoint' commands
+
+This sections describes the `calicoctl endpoint` commands.
```

Comment:
Good opportunity to describe what an endpoint is.  "In Calico an endpoint is a virtual interface from a workload (container or virtual machine) into the Calico network, and workloads may have more than one endpoint.  Calico associates network policy with endpoints."